### ISSUE TITLE
embench fails on chip.eng.hmc.edu

### ISSUE THREAD

**Issue by davidharrishmc** [openhwgroup/cvw#769]
```
$ cd $WALLY/benchmarks/embench
$ make clean
$ make run

...
wikisort
All benchmarks built successfully
../../addins/embench-iot/benchmark_size.py --builddir=bd_speedopt_size --json-output > wallySpeedOpt_size.json 
Traceback (most recent call last):
  File "/home/harris/cvw/benchmarks/embench/../../addins/embench-iot/benchmark_size.py", line 26, in <module>
    import lief
ModuleNotFoundError: No module named 'lief'
make: *** [Makefile:71: size] Error 1
```

**Comment by slmnemo** [comment]
seems like pip install lief would fix this.

**Comment by slmnemo** [comment]
seems sudo -H pip install lief should install it to the home directory of the system instead of the home directory of the user. try that?

**Comment by davidharrishmc** [comment]
Closed with #776 per Kaitlin's -H approach.
